public/css/app.css

```diff
@@ -9330,7 +9330,7 @@ html, body {
 }
 
 .dropdown:hover .dropdown-menu {
-  /*  max-height: 350px; */
+  max-height: 350px;
```

Comment:
This line restricts the vertical height of the dropdown menus resulting in a poor user experience. I commented this out while I was doing some troubleshooting and then never deleted the line. If you see a need to restrict the max height, then we'll also need to adjust the number of elements in the menus because this will cause 2-3 of them to be cut off from view. Please let me know what you are thinking here and what problem you are trying to solve. Thanks!

Reply:
So the issue here was that the `max-height` is set above to `0px` for the slide down animation. Then this is changed to `350px` so the dropdown can slide down. Removing this caused the slide down to be stuck in position (it had something to do with the Javascript that was also doing this broke with the Bootstrap JS upgrade I believe, and this is a simpler and more efficient way of creating the animation - but this is why I included this change in this PR).

Another solution would be to set this to `100vh` so there really is no restriction, or remove the scroll down animation all together, and just have it fade in.

Reply:
Concur, and upon testing I see no unintended consequences of this change. No problem - let's keep the max-height: 350px